packages/shared/routes/setup/ledger/views/GenerateNewAddress.svelte

```diff
@@ -18,16 +18,18 @@
     $: illustration = confirmed ? 'ledger-generate-address-success-desktop' : 'ledger-generate-address-desktop'
 
     function generateNewAddress() {
-        busy = true
         newAddress = null
+        busy = true
 
-        const _onConnected = () =>
+        const _onConnected = () => {
             api.getAccounts({
                 onSuccess(getAccountsResponse) {
                     // If we have already created an account, just get the first address of the first account
                     if (getAccountsResponse.payload.length > 0) {
                         newAddress = getAccountsResponse.payload[0].addresses[0].address
                         busy = false
```

Comment:
@maxwellmattryan perhaps we could set `busy = false` after `displayAddress()` like 
```javascript
function displayAddress() {
        api.getMigrationAddress(true, {
            onSuccess() {
                busy = false
                handleConfirmClick()
            },
            onError(err) {
                newAddress = null
                busy = false
                console.error(err)
            },
        })
    }
```
?

Reply:
Yes, definitely. Thank you for pointing this out!